src/V3Timing.cpp

```diff
@@ -550,7 +550,10 @@ class TimingControlVisitor final : public VNVisitor {
             ss << '"';
             V3EmitV::verilogForTree(sensesp, ss);
             ss << '"';
-            auto* const commentp = new AstCExpr{sensesp->fileline(), ss.str(), 0};
+            // possibly a multiline string
+            std::string comment = ss.str();
+            comment.erase(std::remove(comment.begin(), comment.end(), '\n'), comment.end());
```

Comment:
I think replacing with space would likely be more readable.

Reply:
Done